starter-core/src/main/java/io/micronaut/starter/feature/messaging/jms/AbstractJmsFeature.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2021 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.starter.feature.messaging.jms;
+
+import io.micronaut.starter.application.ApplicationType;
+
+public abstract class AbstractJmsFeature implements JmsFeature {
+
+    @Override
+    public boolean supports(ApplicationType applicationType) {
+        return true;
+    }
+
+    @Override
+    public String getMicronautDocumentation() {
+        return "https://micronaut-projects.github.io/micronaut-jms/latest/guide/index.html";
```

Comment:
This link is unreachable and results in a 404 error.

Reply:
It should point to the snapshot URL. Also, the feature should be marked as preview